### PR TITLE
[Bugfix] Fix lgtm alerts

### PR DESCRIPTION
**Pull request by frogman544**
Fix the lgtm alerts

### CODE REVIEW

File: mobile/src/main/java/de/duenndns/ssl/MemorizingTrustManager.java [hunk 1]
```diff
@@ -418,7 +419,7 @@ private static boolean isPathException(Throwable e) {
     public void checkCertTrusted(X509Certificate[] chain, String authType, boolean isServer)
             throws CertificateException
     {
-        LOGGER.log(Level.FINE, "checkCertTrusted(" + chain + ", " + authType + ", " + isServer + ")");
+        LOGGER.log(Level.FINE, "checkCertTrusted(" + Arrays.toString(chain) + ", " + authType + ", " + isServer + ")");
```
[the remark this answers is not in the document]
Comment (mueller-ma) [reply mobile/src/main/java/de/duenndns/ssl/MemorizingTrustManager.java]: This issue should be fixed upstream: https://github.com/ge0rg/MemorizingTrustManager/issues/57
Using your solution produces a string that it too large to be shown in the log message.

File: mobile/src/main/java/org/openhab/habdroid/ui/LogActivity.java [hunk 2]
```diff
@@ -127,6 +129,7 @@ protected String doInBackground(Boolean... clear) {
                     logBuilder.append(line);
                     logBuilder.append(separator);
                 }
+                bufferedReader.close();
```
Comment (mueller-ma) [inline mobile/src/main/java/org/openhab/habdroid/ui/LogActivity.java]: The reader won't be closed if an exception is thrown. Please see https://lgtm.com/rules/10030075/ for the correct way to fix this issue.

Comment (frogman544) [reply mobile/src/main/java/org/openhab/habdroid/ui/LogActivity.java]: Thank you for the fix! I implemented as described in the rules.

**Comment by mueller-ma** [comment]
I just learned about try-with-ressources which is way more readable: http://tutorials.jenkov.com/java-exception-handling/try-with-resources.html.
Please make sure your android studio doesn't change the import order.

**Comment by mueller-ma** [comment]
@frogman544 I pushed some minor improvements to your branch. Thanks for your contribution.

**Pull request merged**